src/utils/manageExternalImage.ts

```diff
@@ -67,3 +77,59 @@ export const deleteImage = async (imageUrl: string) => {
 
   s3.deleteObject({ Bucket: albumBucketName, Key: photoKey });
 };
+
+export const toBase64 = (file: File) =>
+  new Promise((resolve, reject) => {
+    const reader = new FileReader();
+    reader.readAsDataURL(file);
+    reader.onload = () => resolve(reader.result);
+    reader.onerror = (error) => reject(error);
+  });
+
+export const inputImageCheck = (
+  img: File,
+  minSizeInKB: number,
+  maxSizeInKB: number,
+  toast: any
+) => {
+  const allowedTypes = [".jpg", ".png", ".jpeg"];
+  const imgPath = img.name;
+  const imgFormat = imgPath.substring(imgPath.indexOf("."));
```

Comment:
1. if the file name contains more than one ".", the format is incorrect
please use _img.type_

2. if I have a .exe file and I change the extension to .png and then upload, will get this error

![image](https://user-images.githubusercontent.com/76163407/202323123-d05f11fe-aea1-41ae-852b-2ac209b66210.png)